app/data_providers/snowplow/config.py

```diff
@@ -18,7 +18,7 @@ class SnowplowConfig:
 
     PROTOCOL: HttpProtocol = 'http' if ENV == ENV_LOCAL else 'https'
 
-    CORPUS_SLATE_SCHEMA = 'iglu:com.pocket/corpus_slate/jsonschema/1-0-1'
+    CORPUS_SLATE_SCHEMA = 'iglu:com.pocket/corpus_slate/jsonschema/2-0-1'
```

Comment:
Shouldnt non-breaking field changes be a minor update? Aka `1-1-1`?

Reply:
That's what I was wondering originally too, but G said it was obviously a breaking change, and [Snowplow agrees](https://docs.snowplowanalytics.com/docs/understanding-tracking-design/versioning-your-data-structures/). :smile: 

![Screenshot from 2022-07-26 14-36-51](https://user-images.githubusercontent.com/1547251/181116665-9466f50f-62ea-4dbe-b546-fac98293130e.png)

Reply:
The only reason it's a breaking change is because the added field `recommended_at` is set with a "required" constraint property in Snowplow, making it _not_ backward compatible with the previous version when that field was missing